helpers.py

```diff
@@ -102,3 +102,152 @@ def downloadCreateDataList(itemList,site,format,dataPath):
 
     except Exception:
         raise Exception(traceback.format_exc())
+
+#============================
+#Amino Percentage based
+# a Sequence and a Letter
+#============================
+def calcAminoPercentage(seq,letter):
+  counter=0
+  for alphabet in seq:
+    if (alphabet==letter):
+      counter=counter+1
```

Comment:
You should use the `count` function of list for that: https://stackoverflow.com/questions/2600191/how-can-i-count-the-occurrences-of-a-list-item